examples/README.md

```diff
@@ -1,42 +1,14 @@
-# Example using Differential Privacy library
 
-In this directory, we give a simple example of how to use the Python Differential
-Privacy library.
+## Curated list of Tutorials and Sample code for PyDP!
 
-## Zoo Animals
 
-There are around 182 animals at Farmer Alex's zoo. Every day, Alex feeds
-the animals as many carrots as they desire. The animals record how many carrots
-they have eaten per day. For this particular day, the number of carrots eaten
-can be seen in `animals_and_carrots.csv`.
+### Tutorials
 
-At the end of each day, Alex often asks aggregate questions about how many
-carrots everyone ate. For example, he wants to know how many carrots are eaten
-each day, so he knows how many to order the next day. The animals are fearful
-that Alex will use the data against their best interest. For example, Alex could
-get rid of the animals who eat the most carrots!
+#### Carrots Demo
 
-To protect themselves, the animals decide to use the Python Differential Privacy
-library to aggregate their data before reporting it to Alex. This way, the
-animals can control the risk that Alex will identify individuals' data while
-maintaining an adequate level of accuracy so that Alex can continue to run the
-zoo effectively.
+This is the first example using the PyDP, about animals using the library to aggregate their data (about how many carrots they have eaten) before reporting it to their owner. More about it [here](carrots_demo/README.md).
```

Comment:
"about animals using the library" ????   we want to give an idea of what the example is about - so can put it like this

"Farmer Alex's zoo has around 182 animals. Every day, Alex feeds the animals as many carrots as they desire. The animals record how many carrots they have eaten per day. At the end of each day, Alex often asks aggregate questions about how many carrots everyone ate. The animals decided to use Differential Privacy Library to preserve their privacy. Learn more about it [here]."